backend/src/main/java/com/bakdata/conquery/models/worker/Worker.java

```diff
@@ -164,6 +164,7 @@ public void addImport(Import imp) {
 	}
 
 	public void removeImport(ImportId importId) {
+		storage.removeImport(importId);
```

Comment:
Ouh, hatte ich das vergessen? Das sollte dann nochin den KW41 rein.
Ich wurde das Löschen aus dem Storage in dem Bucketmanager machen. Und da erst die Buckets und Concepts auf räumen und dann erst aus dem Storage löschen.